src/test/java/com/baidu/hugegraph/unit/RestResultTest.java

```diff
@@ -409,54 +404,30 @@ public void testReadVertices() {
                 + "\"label\": \"person\"," 
                 + "\"type\": \"vertex\"," 
                 + "\"properties\": {" 
-                + "\"city\": [{"
-                + "\"id\": \"person:marko>city\"," 
-                + "\"value\": \"Beijing\"" 
-                + "}],"
-                + "\"name\": [{"
-                + "\"id\": \"person:marko>name\"," 
-                + "\"value\": \"marko\"" 
-                + "}],"
-                + "\"age\": [{"
-                + "\"id\": \"person:marko>age\"," 
-                + "\"value\": 29" 
-                + "}]}"
+                + "\"city\": \"Beijing\","
+                + "\"name\": \"marko\","
+                + "\"age\": 29"
+                + "}"
                 + "},"
                 + "{"
                 + "\"id\": \"software:lop\"," 
                 + "\"label\": \"software\"," 
                 + "\"type\": \"vertex\"," 
                 + "\"properties\": {" 
-                + "\"price\": [{"
-                + "\"id\": \"software:lop>price\"," 
-                + "\"value\": 328" 
-                + "}],"
-                + "\"name\": [{"
-                + "\"id\": \"software:lop>name\"," 
-                + "\"value\": \"lop\"" 
-                + "}],"
-                + "\"lang\": [{"
-                + "\"id\": \"software:lop>lang\"," 
-                + "\"value\": \"java\"" 
-                + "}]}"
+                + "\"price\": 328,"
+                + "\"name\": \"lop\","
+                + "\"lang\": \"java\""
+                + "}"
                 + "},"
                 + "{"
                 + "\"id\": \"person:peter\"," 
                 + "\"label\": \"person\"," 
                 + "\"type\": \"vertex\"," 
                 + "\"properties\": {" 
-                + "\"city\": [{"
-                + "\"id\": \"person:peter>city\"," 
-                + "\"value\": \"Shanghai\"" 
-                + "}],"
-                + "\"name\": [{"
-                + "\"id\": \"person:peter>name\"," 
-                + "\"value\": \"peter\"" 
-                + "}],"
-                + "\"age\": [{"
-                + "\"id\": \"person:peter>age\"," 
-                + "\"value\": 29" 
-                + "}]}"
+                + "\"city\": \"Shanghai\","
+                + "\"name\": \"peter\","
+                + "\"age\": 29"
+                + "}"
```

Comment:
add test for list/set property

Reply:
now list/set property will be parsed into a list